utils/ratelimits.py

```diff
@@ -47,11 +47,20 @@ def set(self, key, value):
         return self.__setitem__(key, value)
 
 
-cache = RatelimitCache()
+normalcache = RatelimitCache()
+
+videocache = RatelimitCache(expire_time=timedelta(0, 30, 0))
```

Comment:
Might be a little much but if you find time, perhaps we could remove the need for two different caches by allowing the `setup` decorator to accept `rate` and `per`. I.e. `rate=1, per=30`. Where per is seconds, rate is the amount of times the endpoint can be accessed in the given time limit. This will make the endpoint ratelimiting more flexible should we want to adjust ratelimits of individual endpoints in future.

Although this is not mandatory, this would be nice to see in imgen, if you're up for tackling this.